client/yarn.lock

```diff
@@ -2244,11 +2244,6 @@
   resolved "https://registry.yarnpkg.com/@justfixnyc/geosearch-requester/-/geosearch-requester-0.0.6.tgz#af3144a8b599e45ad3ce5c2190fc60da81106475"
   integrity sha512-wHsY9DbiQUp7H/KiQ4+ejTShR6lY3iagiFhw4VoP1kljyem+WIwEkiZNnvUHkOAQIkTtPulLaiKHEMpzaw9xZQ==
 
-"@justfixnyc/react-common@^0.0.12":
```

Comment:
Huh, so will this obviate the need for react-common across TP *and* WOW? is that the only thing react-common was doing??

Reply:
Heh, for now it would, yeah.  The original goal was for `react-common` to contain all common JustFix-specific react components shared between all our products, but the only thing in there at the moment is the Covid Moratorium Banner.  But perhaps if we have a new need for a shared React component it might come back.  (We could also just have separate NPM packages for each individual component of course--I just find creating new NPM packages w/ their concomitant transpilation build pipelines + test suite configurations + etc to be a big hassle, even with our monorepo's DRY enhancements, so I decided to just put all the components in one package and let tree-shaking excise whatever a project decided not to reuse.)